lib/Linter.js

```diff
@@ -612,7 +652,7 @@ module.exports = (function() {
      *      or null if not.
      * @private
      */
-    function parse(text, config, filePath) {
+    parse(text, config, filePath) {
```

Comment:
I'm thinking whether we should have naming style for private members.
For example, "the name of private members should be symbol", "the name of private members should start with `_`", or etc...
Related to our semver policy.